src/HydraAuction/OnChain/Common.hs

```diff
@@ -30,12 +30,20 @@ minAuctionFee = 2_000_000
 {-# INLINEABLE stageToInterval #-}
 stageToInterval :: AuctionTerms -> AuctionStage -> Interval POSIXTime
 stageToInterval terms stage = case stage of
-  AnnouncedStage -> to (biddingStart terms)
-  BiddingStartedStage -> interval (biddingStart terms) (biddingEnd terms)
-  BiddingEndedStage -> interval (biddingEnd terms) (voucherExpiry terms)
-  VoucherExpiredStage -> interval (voucherExpiry terms) (cleanup terms)
+  AnnouncedStage -> strictTo (biddingStart terms)
+  BiddingStartedStage -> rightExclusiveInterval (biddingStart terms) (biddingEnd terms)
+  BiddingEndedStage -> rightExclusiveInterval (biddingEnd terms) (voucherExpiry terms)
+  VoucherExpiredStage -> rightExclusiveInterval (voucherExpiry terms) (cleanup terms)
   CleanupStage -> from (cleanup terms)
 
+{-# INLINEABLE strictTo #-}
+strictTo :: a -> Interval a
```

Comment:
Move this stuff to utils, please.

Reply:
Where, we don't really have onchain module in utils. I thought common onchain code should live in `onchain/common`